build/BasicTasks.fs

```diff
@@ -6,6 +6,8 @@ open Fake.DotNet
 open Fake.IO.Globbing.Operators
 open System.IO
 open Fake.IO
+open ARCTokenization.StructuralOntology
```

Comment:
While it seems to work, i really do not like this circular dependency, where the build project is dependent on one of the projects it is supposed to build.